tools/accuracy_checker/accuracy_checker/postprocessor/brats_postprocessing.py

```diff
@@ -0,0 +1,128 @@
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import numpy as np
+from scipy.ndimage import interpolation
+from .postprocessor import Postprocessor
+from ..config import ConfigError, BoolField, ListField
+from ..representation import BrainTumorSegmentationPrediction, BrainTumorSegmentationAnnotation
+
+
+def resample(data, shape):
+    if len(data.shape) != len(shape):
+        raise RuntimeError('Dimensions of input array and shape are different. Resampling is impossible.')
+    factor = [float(o) / i for i, o in zip(data.shape, shape)]
+    return interpolation.zoom(data, zoom=factor, order=1)
+
+
+class SegmentationPredictionResample(Postprocessor):
+    __provider__ = "segmentation_prediction_resample"
+
+    prediction_types = (BrainTumorSegmentationPrediction, )
+    annotation_types = (BrainTumorSegmentationAnnotation, )
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'make_argmax': BoolField(optional=True, default=False,
+                                     description="Applies argmax operation for prediction")
+        })
+        return parameters
+
+    def configure(self):
+        self.make_argmax = self.config.get('make_argmax')
+
+    def process_image(self, annotations, predictions):
+        raise RuntimeError("Since `process_image_with_metadata` is overriden, this method MUST NOT be called")
+
+    def process_image_with_metadata(self, annotation, prediction, image_metadata=None):
+        if not len(annotation) == len(prediction) == 1:
+            raise RuntimeError('Postprocessor {} does not support multiple annotation and/or prediction.'
+                               .format(self.__provider__))
+
+        if annotation[0].box:
+            box = annotation[0].box
+        elif image_metadata['box'] is not None:
+            box = image_metadata['box']
+        else:
+            raise ValueError('Postprocessor {} not found bounding box.'.format(self.__provider__))
+
+        annotation_ = annotation[0]
+        prediction_ = prediction[0]
+
+        low = box[0, :]
+        high = box[1, :]
+        diff = (high - low).astype(np.int32)
+
+        annotation_shape = annotation_.mask.shape
+        prediction_shape = prediction_.mask.shape
+
+        image_shape = annotation_shape[len(annotation_shape)-3:len(annotation_shape)]
+        box_shape = (diff[0], diff[1], diff[2])
+
+        label = np.zeros(shape=(prediction_shape[0],) + image_shape)
```

Comment:
if `(prediction_shape[0],) + image_shape` does not change, i think better to save it in the variable and use in all places, where it is necessary

Reply:
`image_shape` is a shape of original data and it can differs from scan to scan